Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: version-patch version-minor version-major version-dry-run credentials clean-credentials list-credentials brew-leaves brew-categorized brew-generate
+.PHONY: version-patch version-minor version-major version-dry-run credentials clean-credentials list-credentials brew-leaves brew-categorized brew-generate test test-verbose test-shell test-setup test-coverage
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)